ts/smelter-web-client/src/manager.ts

```diff
@@ -39,10 +43,23 @@ class RemoteInstanceManager implements SmelterManager {
   public async setupInstance(opts: SetupInstanceOptions): Promise<void> {
     await retry(async () => {
       await sleep(500);
-      return await this.sendRequest({
+      let status = (await this.sendRequest({
```

Comment:
it is only used once in this packaged, but I would extract it to separate function too